docs/docs/sourcing-from-agilitycms.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Sourcing from Agility CMS
+---
+
+This guide takes you through the steps involved in setting up a Gatsby site that fetches content from [Agility CMS](https://agilitycms.com/).
+
+## What is Agility CMS? What makes it different?
+
+[Agility CMS](https://agilitycms.com/) is a headless Content Management System (CMS) that let's you easily define your custom content types and relationships easily.  This is called Content Architecture, and you can reuse this content for your websites and apps.
+
+In addition, Agility CMS provides a page routing API, which allows you to offload control of the sitemap to the content editors.
+
+All content is available through the Agility CMS Fetch or Preview API.
+
+
+## Getting Started
+
+### Create A Free Agility Account
+It only takes a minute to create an Agility CMS account, and it's free forever. [Sign up here](https://account.agilitycms.com/sign-up?product=agility-free).  
+
+Once your account is created, we'll need to grab your GUID and API Keys.
+
+### Get the Code
+
+Make sure you have Gatsby CLI tools installed (we are using npm here...)
+```
+npm install -g gatsby-cli
+```
+
+Go ahead and clone the Agility CMS Gatsby Starter repo on from GitHub that has all the code you need to get started.
+```
+git clone https://github.com/agility/agility-gatsby-starter.git
+```
+
+Resolve any dependencies
+
+```
+npm install
+```
+
+Run it in development mode!
+```
+gatsby develop
+```
+
+The site is just a starter, but it has a bunch of interesting features that you can use to build from.  Let's hook this code up to your new Agility CMS instance that you just created.
+
+## Hook it up to your Agility CMS instance
+Edit the **gatsby-config.js** file and repace the *guid* and *apiKey* with yours.  
```

Comment:
```suggestion
Edit the `gatsby-config.js` file and repace the `guid` and `apiKey` with yours.  
```

Reply:
Are the keys public or should we suggest using secret environment variables? (Also then in readme of the source-plugin)